components/elements/buttons/SocialButtons/SocialButtons.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+interface NavLinksProps {
+  socials: Socials[]
+}
+
+defineProps<NavLinksProps>()
+</script>
+
+<template>
+  <div class="social-buttons">
+    <a
+      v-for="social in socials"
+      :key="social.name"
+      class="social-button"
+      :alt="social.name"
+      :href="social.url"
+    >
+      <img :src="'/icons/social/' + social.icon + '.svg'" :alt="social.icon" />
```

Comment:
can we convert this to a [template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)?